admin/permissions/DDPCICDRole.yaml

```diff
@@ -1,5 +1,4 @@
 description: 'Enable deployments of DDP apps and services on GAE'
-etag: BwWgSLxPGqs=
```

Comment:
@MocanaAtBroad what is best practice with capturing `etag` here?

Reply:
An FYI on how I generated this file: I created original role interactively in gcloud console and then I got the definition in yaml format like so:
`gcloud iam roles describe --project broad-ddp-test ROLENAME > myRoleDefintion.yaml`

The yaml file can be used when calling `gcloud iam roles create` or `gcloud iam roles update`
The etag came from original `describe` invocation.
I believe that it is not an required param.